### DISCUSSION TITLE
usiing models downloaded with ollama

### DISCUSSION THREAD

**Discussion by bokey007** [mudler/LocalAI#1582] [General]
Hi I have been using the combination of ollama + litellm earlier

However, I have recently stumbled across LocalAI and I am exploring it.

since I was using Ollam earlier, I have downloaded many big models using Ollama and therefore I was wondering if I can reference those previously downloaded models in local AI? if yes How?

**Comment by BradKML** [comment]
Unfortunately this sounds like a new feature that might need to get implemented since people see Ollama-LiteLLM as an alternative to LocalAI (e.g. Dify), would be sweet to see a bridge between the two or at the very least model-parity
